server/jetstream_benchmark_test.go

```diff
@@ -335,6 +338,10 @@ func BenchmarkJetStreamConsume(b *testing.B) {
 								b.Fatalf("Error creating stream: %v", err)
 							}
 
+							// reassign connection to stream leader
+							connectURL = cl.streamLeader("$G", streamName).ClientURL()
```

Comment:
nil pointer dereference for N=1

Reply:
Should probably not overwrite nc and js (or if you do close the old one first)

Reply:
@mprimi, I fixed both issues for JetStream(Consume|Publish) benchmarks, ready for another look.